handlers.go

```diff
@@ -4,44 +4,102 @@ import (
 	"fmt"
 	"net/http"
 
+	"encoding/json"
 	"github.com/Financial-Times/go-fthealth/v1a"
-	"regexp"
+	log "github.com/Sirupsen/logrus"
```

Comment:
what would you suggest as an alternative? I think logging conventions in general are different across the teams